.github/CONTRIBUTING.md

```diff
@@ -5,189 +5,297 @@ If you at any point have any questions please read: [I have a question related t
 
 ## Setup environment and tooling for contribution
+
 <!-- The following section is responsible for teaching people how to get setup with 
 the repo such that they can get coding -->
+This section will help you get going with Kirby such that you can begin contributing. 
+
+### Clone and install the repository
+
+<!-- how to clone and get going with the kirby repo-->
+**Note:** make sure you have the right version of Node installed before installing the repository (see: [Node](#node)).
+
+1. Clone the repository: 
+
+``` 
+
+git clone https://github.com/kirbydesign/designsystem
+```
+
+2. Install dependencies: 
 
-### Clone and install repository 
-- how to clone and get going with the kirby repo
+``` 
+
+npm install
+```
+
+3. Start the cookbook server:
+
+``` 
+
+npm start
+```
+
+You should now be able to open `http://localhost:4200` in a browser to access the cookbook.
 
 ### Node 
-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. 
+
+<!-- Which version should people use, and perhaps a recommendation on which tool people can use to manage versions. -->
+[Node](https://nodejs.org/en/) is the runtime used for development. 
+
+It is recommended to use `v12` for development as we currently do not support node LTS (see issue: [#1443](https://github.com/kirbydesign/designsystem/issues/1443)). 
+
+If you need to have multiple versions of node installed then a version manager such as [NVM](https://github.com/nvm-sh/nvm) can be used. 
 
 ### Tslint 
-- People should make sure to have it enabled to avoid unnecessary comments during code review
+
+<!-- People should make sure to have it enabled to avoid unnecessary comments during code review-->
+**Note:** Work is being done to migrate from TSLint to ESLint (see issue: [#1317](https://github.com/kirbydesign/designsystem/issues/1317)). 
+
+TSLint is used for linting which helps keep up the quality of the submitted code for Kirby. 
+
+Scripts for running TSLint can be found in `package.json` . We however recommend that you install a TSLint plugin for your editor. It can be useful to enable "fix on save"-functionality such that simple errors are fixed automatically. 
+
+**Useful resources:**
+
+* [TSLint extension for VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin).
+* [Enabling "fix on save" in VSCode](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin#auto-fix-and-auto-fix-on-save).
+
+#### VSCode "workspace library execution"-issue
+
+An issue might occur where TSLint will randomly disable for the current workspace and has to be enabled again using the command `TSLint: Manage workspace library execution` . 
+
+To solve this: 
+
+1. open a typescript file in VScode. 
+2. open the command pallette (<kbd>ctrl/cmd</kbd> + <kbd>shift</kbd> + <kbd>p</kbd>) and enter: <br />
+
+`typescript: select typescript version` .
```

Comment:
This look a bit odd when displayed. I'm not sure how it's handled in Markdown but I think this line (and similar lines) should be indented, since they're inside a list.

Reply:
Couldn't really get markdown to play nice - so I've inlined them. I think it looks _good enough_:
![Screenshot 2021-05-04 at 12 54 17](https://user-images.githubusercontent.com/17269698/116993492-d798e600-acd7-11eb-93d9-c37f6e2083f9.png)

Reply:
Alright. When I reviewed it it didn't look as nice... 🤷‍♂️. But it looks fine in the screenshot above 👍